cumulusci/cli/cci.py

```diff
@@ -217,26 +218,44 @@ def main(args=None):
         try:
             cli(standalone_mode=False)
         except click.Abort:  # Keyboard interrupt
-            click.echo("\nAborted!")
+            show_debug_info() if debug else click.echo("\nAborted!")
```

Comment:
This is a change in behavior to enter pdb if the user hits Cmd+C while running a command that they ran with the `--debug` flag. It could be annoying or useful; I'm in favor of trying it.